setup-ceph.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+cd /ceph
+git clone https://github.com/openattic/ceph .
```

Comment:
Again, this is very specific to our project development work. Not sure if the scope of this project should be that narrow, or if this should be more versatile.

Reply:
This seems to be a mistake. I should have removed the `git clone` as this project (unlike my fork) clones the repository before the container is started.